src/gitchangelog/gitchangelog.py

```diff
@@ -1290,14 +1293,17 @@ def render_commit(commit: str) -> tuple:
         """
         section = "other"
 
-        # Get Jira summary
+        # Get Jira info
         ticket = None
         if RE_TICKET:
             ticket = RE_TICKET.search(commit["subject"])
             ticket = ticket.group()[1:-1] if ticket else None
         if ticket:
             try:
-                issue = jira.issue(ticket, fields="summary,issuetype")
+                fields = "summary,issuetype"
+                if entry_desc == "jira":
```

Comment:
Why don't you use an enum for the `entry_desc` ? A bit more safe.

Reply:
:+1: Added enum class for entry types

Reply:
You missed his if, 
```suggestion
                 if entry_desc == EntryType.jira:
```